tribits/examples/TribitsSimpleExampleApp/README.md

```diff
@@ -1,10 +1,13 @@
 # TribitsSimpleExampleApp
 
-Simple example project `TribitsSimpleExampleApp` is a raw CMake project that
-pulls in libraries from a few packages from `TribitsExampleProject` using just
-`find_package(TribitsExProj REQUIRED COMPONENTS ...)`.
+This simple example project `TribitsSimpleExampleApp` is a raw CMake project
+that pulls in libraries from a few packages from
+[`TribitsExampleProject`](../TribitsExampleProject/README.md) using just
+`find_package(TribitsExProj REQUIRED COMPONENTS ...)` and also links a smaller
+program using a subset of the libraries from one of the packages from
+`TribitsExampleProject`.
 
-After building and installing TribitsExampleProject under
+After building and installing `TribitsExampleProject` under
```

Comment:
Can be changed by the link :
[`TribitsExampleProject`](../TribitsExampleProject/README.md)

Reply:
Same comment as [above](https://github.com/TriBITSPub/TriBITS/pull/440#discussion_r820942054).